inst/WORDLIST

```diff
@@ -25,7 +25,10 @@ DatasetGroups
 DatasetOrder
 Datetime
 DatetimeVariable
+Dichotomize
```

Comment:
Since you're having this in your diff, I think that means you need to merge master into your branch.